hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationMonitor.java

```diff
@@ -83,9 +83,7 @@ public InvocationMonitor(InvocationRegistry invocationRegistry, ILogger logger,
 
     private long initSlowInvocationThresholdMs(GroupProperties props) {
         long thresholdMs = props.getMillis(GroupProperty.SLOW_INVOCATION_DETECTOR_THRESHOLD_MILLIS);
-        if (thresholdMs > -1) {
-            logger.info("Slow invocation detector enabled, using threshold: " + thresholdMs + " ms");
-        }
+        logger.info("Slow invocation detector enabled, using threshold: " + thresholdMs + " ms");
```

Comment:
I'm not sure if this check should be removed, since `-1` means disabled for `SLOW_INVOCATION_DETECTOR_THRESHOLD_MILLIS`, not `0`, see the definition:
> The default is -1 indicating there is no detection.

So actually a negative value should be fine here.